composer.json

```diff
@@ -6,7 +6,8 @@
     "license":      "Artistic-1.0",
 
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
+        "phpunit/phpunit": "~4.5"
```

Comment:
Good point. I put it in require-dev so there's something there compatible with the existing unit tests.